scan/github_actions_reporter.py

```diff
@@ -133,18 +132,19 @@ def post_comments_to_pull_request(token, repo, pull_number, commit_sha, comment,
 
     # Send the POST request to GitHub API
     headers = {
-        "Authorization": f"token {token}",
+        "Authorization": f"Bearer {token}",
         "Accept": "application/vnd.github.v3+json"
     }
 
     # API URL for pull request comments
     url = f"https://api.github.com/repos/{repo}/pulls/{pull_number}/comments"
 
     logger.debug(f"Building URL to comments POST: {url}")
+    logger.debug(f"Using arguments: {data}")
 
     response = requests.post(url, headers=headers, json=data)
 
-    if response.status_code == 201 or response.status_code == 422:
+    if response.status_code == 201:
```

Comment:
removing 422 to allow more logging in the else case when a validation error has occurred.